### DISCUSSION TITLE
Building with gcc

### DISCUSSION THREAD

**Discussion by ntrel** [Neat-Lang/neat#18] [Q&A] [answered]
Hi,
I'd like to build the compiler on Linux Mint, ideally without LLVM (I've heard it's faster to build projects with gcc and llvm is probably bigger too). I cloned the git repo and then tried:

https://neat-lang.github.io/getstarted.html#install-with-gcc

> Build the compiler:
> ./build.sh

The build.sh script in git seems to need llvm. Is there a way to do it without llvm?

**Comment by FeepingCreature** [comment, marked as answer]
The install instructions assume that you're building one of the releases from the [release page](https://github.com/Neat-Lang/neat/releases). The build script for the releases is generated and specific to the target compiler; it is not identical to the build script in the repo. The build.sh in the repo does assume it's building with LLVM, as that's the primary backend I develop for. (Though the testsuite is run with each backend while building the releases.)

If you want to build the master version with `gcc` , try adding `-backend=c` to the `build.sh`compiler flags. However, note that I don't guarantee that the C backend isn't broken *between* releases.
